.github/workflows/weasel.yml

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Weasel License checks
+
+on:
+  - push
+  - create
+  - pull_request:
+    - opened
+    - reopened
+    - edited
```

Comment:
I am getting
```
Invalid type for `on`
```

Reply:
I think that's because it should have been `created` instead of `create`. Should be fixed.

Reply:
Still getting <code>Invalid type for `on`</code> at b3def6e1cd.

Reply:
Okay, I got it working. For real this time, tested and everything. Even added a badge to the README.